package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "specmatic",
+  "name": "specmatic-Environment",
```

Comment:
Why this change? Is this by mistake?

Reply:
To identify uniquely when i do npm link , and not get confused with the specmatic package in my node_modules. Will remove when the code is final and ready to merge